Helper/Utils.php

```diff
@@ -0,0 +1,299 @@
+<?php
+
+
+namespace Midtrans\Snap\Helper;
+
+
+class Utils
+{
+
+    /**
+     * Replace string for items name
+     *
+     * @param $str
+     * @return string|string[]|null
+     */
+    public function repString($str)
```

Comment:
what's the goal of this function? I guess it's to replace items name so that it become safe/valid value for Midtrans charge API param?, hmm then at least include that explanation on the DocString/DocBlock/DocComment. Because the function name itself is not so descriptive, I don't know what's its for.